pennylane/ops/qubit/parametric_ops.py

```diff
@@ -119,6 +119,10 @@ def _controlled(self, wire):
         new_op = CRX(*self.parameters, wires=wire + self.wires)
         return new_op.inv() if self.inverse else new_op
 
+    def simplify(self):
+        theta = self.data[0] % (4 * np.pi)
+        return RX(theta, wires=self.wires)
```

Comment:
We could return the `Identity` when `theta = 0`.

Reply:
Rather than returning the identity, would it be possible just not to return any gates? On some devices, an identity might be more expensive than nothing

Reply:
If the `Identity` can be removed the simplify pipeline will remove it. But in some cases we want to keep the operator:
```pycon
>>> op_sum = qml.PauliX(0) + qml.RX(2 * np.pi, 0)
>>> op_sum.simplify()
PauliX(wires=[0]) + Identity(wires=[0])
```